lab_1/main.py

```diff
@@ -1,16 +1,24 @@
-"""
-Lab 1
+"""Lab 1
 Language detection
 """
 
-
 def tokenize(text: str) -> list or None:
     """
     Splits a text into tokens, converts the tokens into lowercase,
     removes punctuation and other symbols from words
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if isinstance(text, str):
```

Comment:
it's better to check arguments first and if they're incorrect, return None at the beginning.